.github/workflows/push_pr.yml

```diff
@@ -5,39 +5,27 @@ on:
       - master
   pull_request:
 jobs:
-  lint:
-    name: lint
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - name: Lint
-        run: echo "linting.."
```

Comment:
WOW

Reply:
😭

Reply:
As a matter of fact, I tried to find a quick & easy action to install `shellcheck` but didn't find any so I ended up stripping it completely.